### PR TITLE
Update CKEditor to 37.0.1

### PR DESCRIPTION
**Pull request by rklf**

### CODE REVIEW

File: packages/ckeditor5-self-request/src/ui/linkactionsview.ts [hunk 1]
```diff
@@ -19,6 +19,15 @@ import '@ckeditor/ckeditor5-ui/theme/components/responsive-form/responsiveform.c
 import '../../theme/linkactions.css';
 
 import unlinkIcon from '../../theme/icons/unlink.svg';
+import { Observable, Subscription } from 'rxjs';
+
+
+type selfRequestFunc = {
```
Comment (LeResKP) [inline packages/ckeditor5-self-request/src/ui/linkactionsview.ts]: duplicated

File: packages/ckeditor5-self-request/src/ui/linkactionsview.ts [hunk 2]
```diff
@@ -245,13 +267,6 @@ export default class LinkActionsView extends View {
 
 		return button;
 	}
-
-	destroy() {
```
Comment (LeResKP) [inline packages/ckeditor5-self-request/src/ui/linkactionsview.ts]: Why are you removing the unsubscribe ?

File: packages/ckeditor5-self-request/src/ui/linkformview.ts [hunk 3]
```diff
@@ -52,10 +53,16 @@ const handledKeyCodes = [
 ];
 
 
-function isHandledKey( keyCode ) {
+function isHandledKey( keyCode: any ) {
 	return handledKeyCodes.includes( keyCode );
 }
 
+type selfRequestFunc = {
```
Comment (LeResKP) [inline packages/ckeditor5-self-request/src/ui/linkformview.ts]: duplicated

File: packages/ckeditor5-self-request/src/ui/linkactionsview.ts (hunk 2 above)
Comment (rklf) [reply packages/ckeditor5-self-request/src/ui/linkactionsview.ts]: I didn't remove it, it got moved to ``public override destroy(): void``, please check [here](https://github.com/gymglish/ckeditor5-build-inline/blob/45146b4ef33f3a0152d9953b7de47826dc661a2a/packages/ckeditor5-self-request/src/ui/linkactionsview.ts#L169-L171)

**Review by LeResKP — APPROVED** [review]

**Pull request merged**